test/transforms.jl

```diff
@@ -951,12 +951,14 @@
 
   @testset "Scale" begin
     # constant column
-    x = fill(3.0, 10)
+    x = fill(Float64(3.0), 10)
```

Comment:
Please erase this change as it doesn't change the results.

Reply:
`3.0` is always `Float64` in Julia. For single precision you write `3.0f0`. Integers behave differently in computers: if you specify the type `Int` in your code, then it will be dependent on the word size, either `Int32` or `Int64`.